core/client-api/src/main/java/io/thestencil/client/spi/builders/VersionBuilderImpl.java

```diff
@@ -0,0 +1,57 @@
+package io.thestencil.client.spi.builders;
+
+/*-
+ * #%L
+ * stencil-client-api
+ * %%
+ * Copyright (C) 2021 - 2022 Copyright 2021 ReSys OÜ
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import io.smallrye.mutiny.Uni;
+import io.thestencil.client.api.VersionBuilder;
+import lombok.NoArgsConstructor;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+@NoArgsConstructor
+public class VersionBuilderImpl implements VersionBuilder {
```

Comment:
Let's not do this using a file. Quarkus is a bit of a different framework that, in reality, optimises images. This means that it cuts all code out that is not used. Any files that are not directly specified are cut out as well. 

Reading from a file for just a timestamp and a version number is a bit overkill. 

**Generate via script the timestamp and version number directly into this file.**